pyangbind/lib/serialise.py

```diff
@@ -499,6 +499,15 @@ def load_ietf_json(
                             val = True
                         else:
                             raise ValueError("Invalid value for empty in input JSON " "key: %s, got: %s" % (ykey, val))
+
+                    if chk._yang_type == "identityref":
+                        # identityref values in IETF JSON may contain their module name, as a prefix,
+                        # but we don't build identities with these as valid values. If this is the
+                        # case then re-write the value to just be the name of the identity that we
+                        # should know about.
+                        if ":" in val:
+                            _, val = val.split(":", 2)
```

Comment:
I think this was intended to be `1`, rather than `2`. That second argument is the maximum number of splits to perform, rather than the maximum number of pieces to split the string into. For example:
```python
>>> 'a,b,c,d,e'.split(',', 2)
['a', 'b', 'c,d,e']
```

Reply:
Thanks, you're right -- fixed.